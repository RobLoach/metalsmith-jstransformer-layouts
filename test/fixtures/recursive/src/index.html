---
layout: _layout.html-minifier.jade
---

<h1>Hello World!</h1>

<p>This is a test.</p>
