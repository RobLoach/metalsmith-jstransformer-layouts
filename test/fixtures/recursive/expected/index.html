<!DOCTYPE html><html lang=en><head><title>Jade</title></head><body><h1>Hello World!</h1><p>This is a test.</p></body></html>